### ISSUE TITLE
sensor.interior_power returns avg value instead of sum

### ISSUE THREAD

**Issue by klatka** [jseidl/hass-magic_areas#63]
`sensor.interior_power` is created by magic home:
```
sensors:
  - sensor.schlafzimmer_rollo_electric_w
  - sensor.wohnbereich_rollo_electric_w
  - sensor.buro_rollo_electric_w
unit_of_measurement: W
friendly_name: Interior Power
device_class: power
```

All of the 3 sensors report 0.6 W -> `sensor.interior_power` should return 1.8 W instead of 0.6 W

**Comment by jseidl** [comment]
Good catch, need to separate which `device_class`es are avg'ed and which are sum'ed.

**Comment by jseidl** [comment]
This is being fixed on #74. Now aggregations now have a list of `device_class` which should use `sum()` instead of `mean()`. Coming up on release 2.0.0
